science/gmsh/Portfile

```diff
@@ -29,8 +29,8 @@ distname            gmsh-${version}-source
 extract.suffix      .tgz
 
 checksums           rmd160  3efe6a2f40181a1a2a69d8ec558c61d10e1d37ee \
-                    sha256  0a4269a133b6c23a3fca5d3b381d73117ea073b3fbb2c867327677df87a679c3 \
-                    size    12765988
+                    sha256  194354f5cfede4ef47cb6f1d2bde294873315a1b34be114564defef88d108adf \
+                    size    12794616
```

Comment:
This cannot possibly be correct, as it is not possible for the rmd160 to have remained identical while the sha256 changed.

Reply:
I've never generated a rmd160 please advise

Reply:
@mf2k I don't have `port` installed on this machine. Can you provide me with the `rmd160` please ? :sweat_smile:

Reply:
The guide also tells you how to generate it. See section 12. 

https://guide.macports.org/#development.creating-portfile

Reply:
If you don't have "port" on your machine, you can't possibly test what you're submitting to us. That's not good.